src/ocspdash/web/blueprints/api.py

```diff
@@ -30,40 +31,41 @@
 def register_location_key():
     """Register a public key for an invited location."""
     # TODO: error handling (what if no invite, what if duplicate name, etc.)
-    print(request.data)
     try:
         unverified_claims = jwt.get_unverified_claims(request.data)
-    except jwt.JWTError:
-        abort(HTTPStatus.BAD_REQUEST, 'malformed JWT')
+    except jwt.JWTError as e:
+        raise InvalidUsage(f'failed to decode JWT: {str(e)}')
```

Comment:
you could also do 
```python
raise InvalidUsage('failed to decode JWT') from e
```
unless the point of this was to get all of that information in the string